phase-5/update-node.md

```diff
@@ -18,7 +18,7 @@ When a new version of Desmos is released, all validators need to update their no
    ```bash
    cd ~/desmos
    git fetch -a 
-   git checkout tags/v0.6.2
+   git checkout tags/v0.8.0
    make install
    ```
    
```

Comment:
Should be `phase-5` in the "Get Reward" command.

Reply:
Fixed :ballot_box_with_check: